Source/Fuse.Maps/MapView.ScriptClass.uno

```diff
@@ -15,7 +15,9 @@ namespace Fuse.Controls
 				new ScriptMethod<MapView>("setBearing", setBearing),
 				new ScriptMethod<MapView>("setTilt", setTilt),
 				new ScriptMethod<MapView>("setZoom", setZoom),
+				new ScriptMethod<MapView>("showAllMarkers", showAllMarkers),
 				new ScriptMethod<MapView>("setMarkers", setMarkers),
+				new ScriptPromise<MapView,string,string>("snapshot", ExecutionThread.MainThread, snapshot, null),
```

Comment:
Nit: Add space after comma in type parameter list, like so: `<MapView, string, string>`.